running-a-nats-service/nats_admin/security.md

```diff
@@ -5,4 +5,61 @@ If you are using the [JWT](/running-a-nats-service/configuration/securing_nats/j
 You can use the [`nsc`](/using-nats/nats-tools/nsc/README.md) CLI tool to manage identities. Identities take the form of nkeys. Nkeys are a public-key signature system based on Ed25519 for the NATS ecosystem.
 The nkey identities are associated with NATS configuration in the form of a JSON Web Token (JWT). The JWT is digitally signed by the private key of an issuer forming a chain of trust. The nsc tool creates and manages these identities and allows you to deploy them to a JWT account server, which in turn makes the configurations available to nats-servers.
 
-You can also use [`nk`](https://github.com/nats-io/nkeys#readme) CLI tool and library to manage keys.
\ No newline at end of file
+You can also use [`nk`](https://github.com/nats-io/nkeys#readme) CLI tool and library to manage keys.
```

Comment:
should we link to the in depth guide?
https://docs.nats.io/running-a-nats-service/nats_admin/security/jwt#subjects-available-when-using-nats-based-resolver

Reply:
Added that link